app/views/settings/verifications/show.html.haml

```diff
@@ -16,7 +16,7 @@
 
   .input-copy.lead
     .input-copy__wrapper
-      %input{ type: :text, maxlength: '999', spellcheck: 'false', readonly: 'true', value: link_to('Mastodon', ActivityPub::TagManager.instance.url_for(@account), rel: 'me').to_str }
+      = copyable_input value: link_to(Rails.application.class.module_parent_name, ActivityPub::TagManager.instance.url_for(@account), rel: :me)
```

Comment:
```suggestion
      = copyable_input value: link_to("Mastodon", ActivityPub::TagManager.instance.url_for(@account), rel: :me)
```

This is a nice little refactoring overall, but I am not sure we should replace "Mastodon" here with this obscure method call. I think it is really not obvious what it does. And while both are highly unlikely, I would consider it much more likely that rails would change the way the application class works than us renaming the whole project.

Reply:
That was genuinely the only fun part of this PR. Updated to go back to string.